master/static/migrations/20211117104045_model_version_user_id.tx.up.sql

```diff
@@ -1,5 +1,9 @@
 UPDATE public.model_versions
 SET
     user_id
-    = (SELECT user_id FROM public.models WHERE model_id = public.models.id)
+    = (
+        SELECT models.user_id
+        FROM public.models
+        WHERE models.model_id = public.models.id
```

Comment:
this trips up the tests https://app.circleci.com/pipelines/github/determined-ai/determined/36852/workflows/f9742bea-0eff-4e51-a677-55429d469a47/jobs/1331087